core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/seo/components/dot-device-selector-seo/dot-device-selector-seo.component.html

```diff
@@ -0,0 +1,89 @@
+<p-overlayPanel #op styleClass="dotDeviceSelector__overlayPanel">
```

Comment:
```suggestion
<p-overlayPanel #op styleClass="dot-device-selector__overlayPanel">
```

I think there is no camelCase in BEM. Also